dist/amd/defaultTranslations/relative.time.js

```diff
@@ -55,6 +55,35 @@ define(['exports'], function (exports) {
         'year_in_plural': 'in __count__ years'
       }
     },
+    es: {
```

Comment:
@EisenbergEffect, another PR in short time with changes in dist - what do You think about the git hook i proposed previously to check that files under dist folder aren't included in non-release commit?